### PR TITLE
NET9 docker images

### PR DESCRIPTION
**Pull request by MrSmoke**
Add .net9 `asp`, `runtime`, `sdk` docker images

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -12,9 +12,24 @@ dotnet:
         image_version: 8.0.11
         image_type: bookworm-slim
         env_version_key: DOTNET_VERSION
+    '9.0/aspnet':
+      data:
+        image_repo: mcr.microsoft.com/dotnet/runtime
```
Comment (happymancoder) [inline manifest.yaml:17]: should this be `aspnet` rather than `runtime`?

File: manifest.yaml [hunk 2]
```diff
@@ -12,9 +12,24 @@ dotnet:
...
+        image_version: 9.0.0
+        image_type: bookworm-slim
+        env_version_key: DOTNET_VERSION
```
Comment (happymancoder) [inline manifest.yaml:20]: Should this be `ASPNETCORE_VERSION`?

File: dotnet-sdk/9.0/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,10 @@
+FROM mcr.microsoft.com/dotnet/sdk:9.0.101-bookwork-slim
```
Comment (happymancoder) [inline dotnet-sdk/9.0/Dockerfile:1]: should this be `bookworm` instead of `bookwork`?

File: dotnet/9.0/aspnet/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+
+# Main image
+FROM mcr.microsoft.com/dotnet/runtime:9.0.0-bookworm-slim
+
+ARG newrelic_agent_version
+
+COPY --from=builder /usr/local /usr/local
+
+ENV DOTNET_VERSION=9.0.0 \
```
Comment (happymancoder) [inline dotnet/9.0/aspnet/Dockerfile:24]: Should this be `ASPNETCORE_VERSION=9.0.0`?

File: dotnet/9.0/aspnet/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+ARG newrelic_agent_version
+
+# Download and extract the new relic agent
+RUN curl -SL --output newrelicagent.tar.gz https://download.newrelic.com/dot_net_agent/previous_releases/${newrelic_agent_version}/newrelic-dotnet-agent_${newrelic_agent_version}_amd64.tar.gz \
+    && tar -ozxf newrelicagent.tar.gz -C /usr/local \
+    && rm newrelicagent.tar.gz
+
+
+# Main image
+FROM mcr.microsoft.com/dotnet/runtime:9.0.0-bookworm-slim
```
Comment (happymancoder) [inline dotnet/9.0/aspnet/Dockerfile:18]: `aspnet:9.0.0-bookworm-slim`?

File: manifest.yaml (hunk 1 above)
Comment (MrSmoke) [reply manifest.yaml]: oops copied and pasted wrong block. fixed

File: manifest.yaml (hunk 2 above)
Comment (MrSmoke) [reply manifest.yaml]: yep fixed

File: dotnet-sdk/9.0/Dockerfile (hunk 3 above)
Comment (MrSmoke) [reply dotnet-sdk/9.0/Dockerfile]: fixed in the manifest

**Pull request merged**